prover/src/main.rs

```diff
@@ -38,6 +38,14 @@ enum Command {
             default_value = "nexus-public.zst"
         )]
         pp_file: String,
+
+        /// whether public parameters are compatible with proof compression
+        #[arg(short = 'c', long = "compressible", default_value = "false")]
+        com: bool,
+
+        /// SRS file: required with `--compressible`, otherwise ignored
+        #[arg(short = 's', long = "srs")]
+        srs_file: Option<String>,
```

Comment:
Why are both needed if `srs_file: None` implies `com: false`? Or you can mark it with `require` from clap docs.